packages/bridge-controller/src/constants/swaps.ts

```diff
@@ -0,0 +1 @@
+export const SWAPS_API_V2_BASE_URL = 'https://swap.api.cx.metamask.io';
```

Comment:
⏭️ The `swaps-controller` repo has similar constants, including this one: https://github.com/MetaMask/swaps-controller/blob/021cf1f3c248189ffb998818eae4f449d46a90c7/src/constants.ts. I wonder if it would be worth depending on `swaps-controller` for this package so we don't have to repeat everything here?